src/main.rs

```diff
@@ -136,11 +136,24 @@ async fn main() -> Result<()> {
         Some("push") => {
             if let Some(ref matches) = matches.subcommand_matches("push") {
                 let (sources, docker_config) = remote_server_options(matches)?;
-                let policy = fs::read(matches.value_of("policy-path").unwrap())?;
-                let uri = matches.value_of("uri").unwrap();
+                let wasm_uri = crate::utils::map_path_to_uri(matches.value_of("policy").unwrap())?;
+                let wasm_path = crate::utils::wasm_path(wasm_uri.as_str())?;
+                let uri = matches
+                    .value_of("uri")
+                    .map(|u| {
+                        if u.starts_with("registry://") {
+                            String::from(u)
+                        } else {
+                            format!("registry://{}", u)
+                        }
+                    })
+                    .unwrap();
+
+                println!("Attempting to push {:?} to {:?}", wasm_path, uri);
```

Comment:
I'd be silent by default, and maybe use some logging library with verbosity, structured logging... I think we can start building on that story.